test/val/val_atomics_test.cpp

```diff
@@ -422,8 +422,12 @@ OpAtomicStore %f32_var_function %device %relaxed %f32_1
   CompileSuccessfully(GenerateShaderCode(body), SPV_ENV_VULKAN_1_0);
   ASSERT_EQ(SPV_ERROR_INVALID_DATA, ValidateInstructions(SPV_ENV_VULKAN_1_0));
   EXPECT_THAT(getDiagnosticString(),
-              HasSubstr("AtomicStore: Function storage class forbidden when "
-                        "the Shader capability is declared."));
+              AnyVUID("VUID-StandaloneSpirv-None-04686"));
```

Comment:
Can you please replicate this test for a universal environment to ensure we still disallow Function for Shader capability generally.